android-project/app/src/main/java/org/libsdl/app/HIDDeviceManager.java

```diff
@@ -584,7 +554,14 @@ public boolean openDevice(int deviceID) {
         if (usbDevice != null && !mUsbManager.hasPermission(usbDevice)) {
             HIDDeviceOpenPending(deviceID);
             try {
-                mUsbManager.requestPermission(usbDevice, PendingIntent.getBroadcast(mContext, 0, new Intent(HIDDeviceManager.ACTION_USB_PERMISSION), 0));
+                final int FLAG_MUTABLE = 0x02000000; // PendingIntent.FLAG_MUTABLE, but don't require SDK 31
+                int flags;
```

Comment:
why not flags = 0 and get rid of else? ;d

Reply:
I just copied the files from upstream 😜